gluecodium/src/main/resources/templates/dart/DartInterface.mustache

```diff
@@ -52,7 +52,11 @@ final __release_handle = __lib.nativeLibrary.lookupFunction<
     void Function(Pointer<Void>)
   >('{{resolveName "Ffi"}}_release_handle');
 
-class {{resolveName}}__Impl implements {{resolveName}}{
+{{#functions}}
```

Comment:
filtering here instead of in the linked template would make this bit easier to understand